__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/test-data/index.js');
+const request = require('supertest');
+const app = require('../app/index.js');
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
+
+describe('API ready', () => {
+	test('200: responds with a msg that the api is available', () => {
```

Comment:
We don't really need an endpoint that responds with a message that the API is available. Please remove this test and associated code before merging to main as it is not needed for this sprint.